python/tk_desktop/grouping_list_view.py

```diff
@@ -114,7 +114,7 @@ class GroupingListView(ActionListView):
     expanded_changed = QtCore.Signal(str, bool)
 
     def __init__(self, parent=None):
-        QtGui.QListView.__init__(self, parent)
+        ActionListView.__init__(self, parent)
```

Comment:
True dat. Wonder why we got into that habit in the first place. You'd think that we would be using super to ease maintenance...